lib/activemodel/all/activemodel_test.rb

```diff
@@ -77,3 +77,24 @@ module ActiveModelTest
   validates_numericality_of :width, greater_than: :minimum_weight, on: [:create, :update]
   validates_presence_of :first_name, on: [:create, :update]
 end
+
+class Person
```

Comment:
can you please add a test where this descends from `ActiveRecord::Base` too?

there were issues with the `ActiveModel::Validations` sigs where they didn't work properly in AR, which a test would have caught - see https://github.com/sorbet/sorbet-typed/pull/216/files#diff-52d2b7591f89a773def6fce9c1ca7ad7R80 for an example.

Reply:
You mean `ApplicationRecord`? I can, but it causes Person not to have a `new` method so I have to write the initializer like `person = T.let(T.unsafe(nil), Person)`

Reply:
@connorshea did you also add this line? https://github.com/sorbet/sorbet-typed/pull/216/files#diff-52d2b7591f89a773def6fce9c1ca7ad7R39

Reply:
@ghiculescu that link doesn't actually refer to a specific line.

Reply:
Indeed it doesn't, and now I can't remember what I was trying to link to :(

Reply:
Oh right. It was this line: https://github.com/sorbet/sorbet-typed/blob/master/lib/activerecord/all/activerecord_test.rb#L39